std-bits/table/src/main/java/org/enso/table/read/DelimitedReader.java

```diff
@@ -42,7 +42,9 @@ public class DelimitedReader {
    * <p>I considered to choose `\u0F8EE` which comes from the Private Use Area of the Basic
    * Multilingual Plane. Is has no meaning designated by the Unicode standard.
    */
-  public static final char UNUSED_CHARACTER = '\0';
+  public static final char COMMENT_CHARACTER = '\0';
+
+  public static final char UNUSED_CHARACTER = (char) 63726; // \u0F8EE
```

Comment:
Any java experts know of a better way to do this? Apparently \u0F8EE gets resolved too early by the compiler...